features/step_definitions/networking.rb

```diff
@@ -975,3 +975,13 @@
   raise "Failed to kill the #{ovndb} database daemon" unless @result[:success]
 end
 
+Given /^OVN is functional on the cluster$/ do
+  ensure_admin_tagged
+  ovnkube_node_ds = daemon_set('ovnkube-node', project('openshift-ovn-kubernetes')).replica_counters(user: admin)
```

Comment:
Most object methods use cached values to avoid multiple API calls. It makes sense here to call `replica_counters(user: admin, cached: false)` to make sure you are getting values at the moment. The next step can still use cached value because we already updated the object cache. No need for a second API call.